enterprise/api/v1/views.py

```diff
@@ -658,6 +665,64 @@ def get_serializer_class(self):
 
         raise ValidationError('No enrollment found for the given fulfillment source uuid.', code=HTTP_404_NOT_FOUND)
 
+    def get_unenrolled_fulfillment_queryset(self):
+        """
+        Return the queryset for this view. Queries across subsidy types until it finds a match for the provided uuid.
+        Returns a 404 if no subsidy fulfillment record is found.
+        """
+        # Adding licensed enrollment support for future implementations
+        if self.request.query_params.get('retrieve_licensed_enrollments'):
+            enrollment_table = models.LicensedEnterpriseCourseEnrollment
+        else:
+            enrollment_table = models.LearnerCreditEnterpriseCourseEnrollment
+
+        unenrolled_queryset = enrollment_table.objects.filter(
+            enterprise_course_enrollment__unenrolled=True,
+        )
+        # Apply a modified filter if one is provided via query params
+        if self.request.query_params.get('modified'):
+            unenrolled_queryset = unenrolled_queryset.filter(
+                enterprise_course_enrollment__modified__gte=self.request.query_params.get('modified')
+            )
+
+        if not self.request.user.is_staff:
+            ent_customer_uuid = get_enterprise_customer_from_user_id(self.request.user.id)
+
+            enterprise_customer_filter = Q(
+                enterprise_course_enrollment__enterprise_customer_user__enterprise_customer__uuid=ent_customer_uuid
+            )
+            unenrolled_queryset = unenrolled_queryset.filter(enterprise_customer_filter)
+
+        return unenrolled_queryset
```

Comment:
It's not necessary for users with the operator role to be marked with `is_staff=true`.  You may just want to remove this filter from the queryset if the endpoint will only ever be called by operators.